core/src/test/java/io/questdb/griffin/SortAndLimitTest.java

```diff
@@ -0,0 +1,195 @@
+package io.questdb.griffin;
+
+import org.junit.Test;
+
+/**
+ *
+ */
+public class SortAndLimitTest extends AbstractGriffinTest {
```

Comment:
please add tests with `order by DESC`

Reply:
please add tests where sorted values are not all distinct

Reply:
I copied some tests from https://github.com/questdb/questdb/pull/1748 and also tested tree creation with duplicates .